### ISSUE TITLE
error during install uvdesk

### ISSUE THREAD

**Issue by noureldinmegahed** [uvdesk/community-skeleton#539]
While installing uvdesk it gave me this error:
![Screenshot 2022-03-10 185639](https://user-images.githubusercontent.com/77926574/157714887-058c4682-e5a5-478b-8f4b-b66a9b1b3aca.png)
How Can me fix this?

**Comment by Komal-sharma-2712** [comment]
@noureldinmegahed 
Can you please tell us which version you are using of UVdesk?

It may be due to **less max_execution_time** set with your **php.ini** file.

You can increase it to 100 by default it should be 30.
Or if you are still facing issue you can try the manual set method as well.

Please check the below link for manual setup:
[https://github.com/uvdesk/community-skeleton/issues/290#issuecomment-661113642](https://github.com/uvdesk/community-skeleton/issues/290#issuecomment-661113642)

**Comment by noureldinmegahed** [comment]
> @noureldinmegahed 
> Can you please tell us which version you are using of UVdesk?
> 
> It may be due to **less max_execution_time** set with your **php.ini** file.
> 
> You can increase it to 100 by default it should be 30.
> Or if you are still facing issue you can try the manual set method as well.
> 
> Please check the below link for manual setup:
> [https://github.com/uvdesk/community-skeleton/issues/290#issuecomment-661113642](https://github.com/uvdesk/community-skeleton/issues/290#issuecomment-661113642)
> 
> 
> 

I use version "1.0.18"

**Comment by noureldinmegahed** [comment]
>@Komal-sharma-2712 

I use version: 1.0.18

**Comment by noureldinmegahed** [comment]
> @Komal-sharma-2712 

I use version: 1.0.18

**Comment by Komal-sharma-2712** [comment]
@noureldinmegahed 
Have you tried with manual setup?

**Comment by noureldinmegahed** [comment]
@Komal-sharma-2712 
Yes, but when I enter the database user password, it does not accept this password + it's give me this error:
![image](https://user-images.githubusercontent.com/77926574/158133959-8d41ea22-34e9-4375-a9a1-12db7858de99.png)

**Comment by papnoisanjeev** [comment]
@noureldinmegahed 

Please check [here](https://github.com/uvdesk/community-skeleton/issues/364#issuecomment-780486976) hope it will help.

**Comment by noureldinmegahed** [comment]
> @papnoisanjeev 

The code that you tell me to make a comment does not exist in the same way
